gptcache/manager/scalar_data/sql_storage.py

```diff
@@ -0,0 +1,204 @@
+from typing import List
+
+from datetime import datetime
+
+from gptcache.utils import import_sqlalchemy
+from gptcache.manager.scalar_data.base import CacheStorage, CacheData
+
+import_sqlalchemy()
+
+from sqlalchemy import func, create_engine, Column, Sequence  # pylint: disable=C0413
+from sqlalchemy.types import (  # pylint: disable=C0413
+        String,
+        DateTime,
+        LargeBinary,
+        Integer,
+    )
+from sqlalchemy.orm import sessionmaker  # pylint: disable=C0413
+from sqlalchemy.ext.declarative import declarative_base  # pylint: disable=C0413
+
+Base = declarative_base()
+
+
+def get_models(table_prefix, db_type):
+    class QuestionTable(Base):
+        """
+        question table
+        """
+
+        __tablename__ = table_prefix + "_question"
+        __table_args__ = {"extend_existing": True}
+
+        if db_type == "oracle":
+            id = Column(
+                Integer, Sequence("id_seq", start=1), primary_key=True, autoincrement=True
+            )
+        else:
+            id = Column(Integer, primary_key=True, autoincrement=True)
+        question = Column(String(6000), nullable=False)
+        create_on = Column(DateTime, default=datetime.now)
+        last_access = Column(DateTime, default=datetime.now)
+        embedding_data = Column(LargeBinary, nullable=True)
+        deleted = Column(Integer, default=0)
+
+    class AnswerTable(Base):
+        """
+        answer table
+        """
+        __tablename__ = table_prefix + "_answer"
+        __table_args__ = {"extend_existing": True}
+
+        if db_type == "oracle":
+            id = Column(
+                Integer, Sequence("id_seq", start=1), primary_key=True, autoincrement=True
+            )
+        else:
+            id = Column(Integer, primary_key=True, autoincrement=True)
+        question_id = Column(Integer, nullable=False)
+        answer = Column(String(1000), nullable=False)
+        answer_type = Column(String(255), nullable=False)
+        create_on = Column(DateTime, default=datetime.now)
+        last_access = Column(DateTime, default=datetime.now)
```

Comment:
`create_on` and `last_access` is not needed, because the cache search corresponds mainly to the problem

Reply:
fixed